libsolidity/interface/SourceReferenceFormatter.cpp

```diff
@@ -46,29 +48,28 @@ void SourceReferenceFormatter::printSourceLocation(
 	int endLine;
 	int endColumn;
 	tie(endLine, endColumn) = scanner.translatePositionToLineColumn(_location->end);
-	if (startLine == endLine)
-	{
-		string line = scanner.lineAtPosition(_location->start);
-		_stream << line << endl;
-		for_each(
-			line.cbegin(),
-			line.cbegin() + startColumn,
-			[&_stream](char const& ch) { _stream << (ch == '\t' ? '\t' : ' '); }
-		);
-		_stream << "^";
-		if (endColumn > startColumn + 2)
-			_stream << string(endColumn - startColumn - 2, '-');
-		if (endColumn > startColumn + 1)
-			_stream << "^";
-		_stream << endl;
-	}
-	else
-		_stream <<
-			scanner.lineAtPosition(_location->start) <<
-			endl <<
-			string(startColumn, ' ') <<
-			"^\n" <<
-			"Spanning multiple lines.\n";
+
+	int width = std::to_string(startLine).size();
+	string line = scanner.lineAtPosition(_location->start);
+
+	auto color = termcolor::red;
+	if (_name == "Warning")
```

Comment:
This definitely has to be a severity bool.

Reply:
Absolutely. I wanted to do it on the cheap for now to gather feedback on the final result first.

Reply:
I was thinking we can add an `Error::Severity` enum and provide a `severityToString` function. Would that work?

Reply:
I guess for this particular case it could be just `bool isWarning` or something like that?

Reply:
I had done some work with a severity enum. I'll rebase and push it now.